mteb/tasks/Classification/vie/VieStudentFeedbackClassification.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import annotations
+
+import random
+from collections import Counter
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+TEST_SAMPLES = 2048
+
+
+class VieStudentFeedbackClassification(AbsTaskClassification):
```

Comment:
2 points for new dataset (+4 bonus for first classification task for the language). We do have a vie task in MassiveIntent, but it is a translation so I would not count it.